README.md

```diff
@@ -89,7 +89,7 @@ public class SimpleAppTracker extends BaseAnalyticsTracker {
             instance = new SimpleAppTracker(application, FabricTrackerImpl.builder(application).build(),
                     FirebaseTrackerImpl.builder(application).setExceptionTrackingEnabled(true).build(),
                     TestfairyTrackerImpl.builder(application, "KEY").build());
-            FirebasePerformance.getInstance().setPerformanceCollectionEnabled(!BuildConfig.DEBUG);
+            FirebasePerformance.getInstance().setPerformanceCollectionEnabled(!BuildConfig.DEBUG && instance.hasEnabledTracker());
```

Comment:
This will add a dependecy to firebase in every build config.
tbh this was/is the reason why it`s not in base tracker

Reply:
The readme mentions explicitly further below, that FirebasePerformance is not part of this framework. This is merely a hint on how one would do the integration (also check out the Firebase Performance section further down in the README).

So this is just part of the documentation. :)

Reply:
omg, sry !!!